### ISSUE TITLE
A color-reverse version of the font is needed

### ISSUE THREAD

**Issue by starover77** [slavonic/fonts-znam#14]
A color-reverse version of the font is needed, for use in polyphonic scores, as well as for explanations of neumes according to the usage of Kalashnikov and an entire manuscript tradition. In this variant font, all base neumes and black marks should appear in red by default, and all red marks should appear in black.

**Comment by starover77** [comment]
Of course, we might need to hold off on this project until we get an approval from the UTC and we are ready to release a stable public version.

**Comment by typiconman** [comment]
Yes. This is pretty straightforward to do.

**Comment by typiconman** [comment]
@starover77 is this still needed? It seems somewhat straightforward to do. Should we call the font Mezenets Inverted or something else?

**Comment by starover77** [comment]
Yes, it is needed for typesetting polyphonic neumatic manuscripts, particularly the 3- and 4-part Strochnoe penie and Demestvenny (etc.) settings. Alternating parts swap colors so that the singers can more readily find their parts.
